cdi-core/src/main/java/com/linkedin/cdi/factory/network/KeyCertRetriever.java

```diff
@@ -58,6 +58,7 @@ public KeyCertRetriever(State state) {
       privateKey = Base64.getEncoder().encodeToString(entry.getPrivateKey().getEncoded());
       cert = Base64.getEncoder().encodeToString(entry.getCertificate().getEncoded());
     } catch (KeyStoreException | NoSuchAlgorithmException | CertificateException | UnrecoverableEntryException e) {
+      LOG.error("Error while loading key and cert for Alias : {}", keyAlias, e);
```

Comment:
Missing `{}`
Check the `LOG.error` api to see if there there is a relevant api. 
Else, break the logging into 2 lines.

Reply:
Since we are using SL4J for logging It should correctly log the message with the keyAlias and include the stack trace of the exception e.
In Log4J this might fail.